tests/e2e/Services/Account/AccountBase.php

```diff
@@ -317,137 +317,137 @@ public function testGetAccountSessions($data): array
         return $data;
     }
 
-    // /**
-    //  * @depends testCreateAccountSession
-    //  */
-    // public function testGetAccountLogs($data): array
-    // {
-    //     sleep(10);
-    //     $session = $data['session'] ?? '';
-    //     $sessionId = $data['sessionId'] ?? '';
-    //     $userId = $data['id'] ?? '';
-    //     /**
-    //      * Test for SUCCESS
-    //      */
-    //     $response = $this->client->call(Client::METHOD_GET, '/account/logs', array_merge([
-    //         'origin' => 'http://localhost',
-    //         'content-type' => 'application/json',
-    //         'x-appwrite-project' => $this->getProject()['$id'],
-    //         'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
-    //     ]));
-
-    //     $this->assertEquals($response['headers']['status-code'], 200);
-    //     $this->assertIsArray($response['body']['logs']);
```

Comment:
Can we delete / uncomment this part ? Seems like we have duplication here

Reply:
It was commented and I enabled it. I don't see any duplications.